src/pages/marketplace/contributors.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Contributors from '../marketplace/top-contributors.md'
+import Layout from '@theme/Layout';
+import styles from "./styles.module.css";
+
+
+function topContrib() {
```

Comment:
why is this needed?

Reply:
So I can't put style easily on the markdown file itself. It better to handle this in a js file where I can control whatever I need and want